content/messaging/references/default-properties/index.mdx

```diff
@@ -209,6 +209,15 @@ Specifies the time and date of the most recent transfer of a chat.
 | **Access**    | `r` for Agents in threads |
 | **Data type** | `int`                     |
 
+### offline_message
+
+Specifies if a thread is an offline message.
```

Comment:
I'm not sure about this definition. A thread consists of messages, so thread != message.

Reply:
Good point.  "Offline message" here is a name of legacy feature that we are porting to LC3 where that message will be represented as a thread.  Can we work with a different wording like, "... thread that was created as an offline message" or something like that?

Reply:
How about: 

> Specifies if a thread that was initially created as an offline message.